stdlib/Sockets/src/Sockets.jl

```diff
@@ -275,7 +275,9 @@ function bind(sock::Union{TCPServer, UDPSocket}, host::IPAddr, port::Integer; ip
             return false
         end
     end
-    sock.status = StatusOpen
+    if typeof(sock) isa TCPSocket
```

Comment:
Shouldn't this be `if isa(sock, TCPServer) || isa(sock, UDPSocket)` instead?
I think CI has failed because of this.

Reply:
Sorry that was a dumb mistake! fixed in 28f5428